ci/testflight/mempool/testflight-values.yml

```diff
@@ -0,0 +1,9 @@
+persistence:
+  enabled: false
+
+bitcoindRpcPort: 18443
+
+mariadb:
```

Comment:
Leftover?

Reply:
thanks addressed in [d049bc5](https://github.com/GaloyMoney/charts/pull/3105/commits/d049bc54c4bd46f83e99f6427d30bd76c9a4fbe5)